core/server/services/auth/api-key/admin.js

```diff
@@ -0,0 +1,100 @@
+const jwt = require('jsonwebtoken');
+const models = require('../../../models');
+const common = require('../../../lib/common');
+
+const JWT_OPTIONS = {
+    maxAge: '5m',
+    algorithms: ['HS256']
+};
+
+/**
+ * Remove 'Bearer' from raw authorization header and extract the JWT token.
+ * Eg. Authorization: Bearer ${JWT}
+ * @param {string} header
+ */
+const extractTokenFromHeader = function extractTokenFromHeader(header) {
+    const [scheme, token] = header.split(' ');
```

Comment:
_might_ be a good idea to split on `/\s+/` here

Reply:
Hmm, I'd be inclined to say the header format should be as strict as possible to avoid any ambiguity. `\s` can match tabs, newlines, and carriage returns which I don't think we want.

Reply:
Fair :+1: